### ISSUE TITLE
[FEATURE] Display site icons for each link in SearchResult

### ISSUE THREAD

**Issue by ryankwirth** [police-brutality-2020/visualizer#15]
## Purpose

Showing icons for incident links in the search results provides a better overview of the available data.

## 💡 Proposed Implementation:

- Create a new `div` list in the `SearchResult` component
- Populate the list with icons from the `parseUrl(...)` hook

**Comment by JKirkYuan** [comment]
@RyanWirth addressed in #33 ?

**Comment by RyanWirth** [comment]
No, I mean in the result list, sort of like the original mockup I made.

**Comment by JKirkYuan** [comment]
Oh right, like the pins for search?

**Comment by RyanWirth** [comment]
Like the Twitter and Instagram icons here:

![Search Results](https://user-images.githubusercontent.com/6540048/84466780-74bddc80-ac37-11ea-9e58-6f784adc8ab2.png)

I'll create a new issue for the search pins.

**Comment by JKirkYuan** [comment]
I'm gonna setup PR & Issue templates :)
